scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -65,7 +65,7 @@ then
 else
   #mirror="https://mirror.math.princeton.edu"
   mirror="https://ftp.osuosl.org"
-  software_version=`doGetVariable ECLIPSE_VERSION 2018-12`
+  software_version=$(doGetVariable ECLIPSE_VERSION 2018-12)
```

Comment:
shouldnt this be changed to `${ECLIPSE_VERSION:-2018-12}`?

Reply:
True (as mentioned within #64 - but since this was not a shellcheck finding, I skipped this change in this pull request.